godot-core/src/docs.rs

```diff
@@ -23,40 +23,56 @@ pub struct StructDocs {
     pub members: &'static str,
 }
 
-/// Created for documentation on
+/// Keeps documentation for Inherent Implementation such as:
 /// ```ignore
 /// #[godot_api]
 /// impl Struct {
 ///     #[func]
 ///     /// This function panics!
 ///     fn panic() -> f32 { panic!() }
+///     #[signal]
+///     /// this signal signals
+///     fn documented_signal(p: Vector3, w: f64);
+///     #[constant]
+///     /// this constant consts
+///     const CON: i64 = 42;
+///
```

Comment:
```suggestion
///     #[signal]
///     /// this signal signals
///     fn documented_signal(p: Vector3, w: f64);
///
///     #[constant]
///     /// this constant consts
///     const CON: i64 = 42;
```
(I just moved the empty line from the end to between them, no clue why the diff is so weird)